src/main/webapp/app/exam/participate/exercises/exam-exercise-update-highlighter/exam-exercise-update-highlighter.component.ts

```diff
@@ -24,10 +25,20 @@ export class ExamExerciseUpdateHighlighterComponent implements OnInit, OnDestroy
 
     ngOnInit(): void {
         this.subscriptionToLiveExamExerciseUpdates = this.examExerciseUpdateService.currentExerciseIdAndProblemStatement.subscribe((update) => {
-            this.updateExerciseProblemStatementById(update.exerciseId, update.problemStatement);
+            if (update) {
+                this.updateExerciseProblemStatementById(update.exerciseId, update.problemStatement);
+                this._isHidden = false;
+            } else {
+                // No update so hide the component
+                this._isHidden = true;
+            }
         });
     }
 
+    get isHidden(): boolean {
+        return this.previousProblemStatementUpdate === undefined || this._isHidden;
+    }
```

Comment:
We should avoid method calls from the template, so please move this piece directly into the `[hidden]` property.

Reply:
Made the necessary changes. Thanks for the review.